.gitignore

```diff
@@ -0,0 +1,3 @@
+test.html
+node_modules
+package.json
```

Comment:
why you add package.json to .gitignore? without this file nobody will build this project ;p

Reply:
little mistake :)